Makefile

```diff
@@ -87,8 +88,14 @@ install-ingest-wikipedia:
 install-unstructured-inference:
 	python3 -m pip install -r requirements/local-inference.txt
 
+.PHONY: install-tensorboard
+install-tensorboard:
+	@if [ ${OS} = "Darwin" ]; then\
+		python3 -m pip install tensorboard;\
+	fi
```

Comment:
I think technically this is breaking on any aarch64 device, but probably Linux ARM are mostly raspberry pi folks atm?

Reply:
Probably, and there are Linux arm based instances on AWS, I have a better way, will push it up in a minute

Reply:
Ok updated to use CPU arch instead of os